reference/supervisor.md

```diff
@@ -1031,6 +1031,48 @@ and setting fields with the above described functions.
 
 **Name**
 
+**wb\_supervisor\_field\_import\_mf\_bool**, **wb\_supervisor\_field\_import\_mf\_int32**, **wb\_supervisor\_field\_import\_mf\_float**, **wb\_supervisor\_field\_import\_mf\_vec2f**, **wb\_supervisor\_field\_import\_mf\_vec3f**, **wb\_supervisor\_field\_import\_mf\_rotation**, **wb\_supervisor\_field\_import\_mf\_color**, **wb\_supervisor\_field\_import\_mf\_string**, **wb\_supervisor\_field\_remove\_mf\_bool**, **wb\_supervisor\_field\_remove\_mf\_int32**, **wb\_supervisor\_field\_remove\_mf\_float**, **wb\_supervisor\_field\_remove\_mf\_vec2f**, **wb\_supervisor\_field\_remove\_mf\_vec3f**, **wb\_supervisor\_field\_remove\_mf\_rotation**, **wb\_supervisor\_field\_remove\_mf\_color**, **wb\_supervisor\_field\_remove\_mf\_string** - *import or remove a value in a field*
+
+{[C++](cpp-api.md#cpp_field)}, {[Java](java-api.md#java_field)}, {[Python](python-api.md#python_field)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_field_import_mf_bool(WbFieldRef field, int position, bool value);
+void wb_supervisor_field_import_mf_int32(WbFieldRef field, int position, int value);
+void wb_supervisor_field_import_mf_float(WbFieldRef field, int position, double value);
+void wb_supervisor_field_import_mf_vec2f(WbFieldRef field, int position, const double values[2]);
+void wb_supervisor_field_import_mf_vec3f(WbFieldRef field, int position, const double values[3]);
+void wb_supervisor_field_import_mf_rotation(WbFieldRef field, int position, const double values[4]);
+void wb_supervisor_field_import_mf_color(WbFieldRef field, int position, const double values[3]);
+void wb_supervisor_field_import_mf_string(WbFieldRef field, int position, const char *value);
+
+void wb_supervisor_field_remove_mf_bool(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_int32(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_float(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_vec2f(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_vec3f(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_rotation(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_color(WbFieldRef field, int position);
+void wb_supervisor_field_remove_mf_string(WbFieldRef field, int position);
```

Comment:
To summarize our skype discussion:
  - We decided to rename the new functions to `wb_supervisor_field_insert_mf_*`
  - Keep the `wb_supervisor_field_import_mf_node*` function as is (and eventually introduce in the future a `wb_supervisor_field_insert_mf_node` function that would take a `WbNodeRef` as argument, @fabienrohrer will open an issue a bout this).
  - Have a unique `wb_supervisor_field_remove_mf` (or `wb_supervisor_field_remove_mf_item`?) function to remove an item from a MF node.
  - Deprecate the `wb_supervisor_field_remove_mf_node` function.